### PR TITLE
feat: add sprig support...continued

### PR DESCRIPTION
**Pull request by talbright**
## What does this do / why do we need it?

Finish what https://github.com/git-chglog/git-chglog/pull/99 started.

* update README.md 
* remove duplicate template functions
* add test

## How this PR fixes the problem?

See https://github.com/git-chglog/git-chglog/pull/99

## What should your reviewer look out for in this PR?

See https://github.com/git-chglog/git-chglog/pull/99

## Check lists

* [x] Test passed
* [x] Coding style (indentation, etc)

## Additional Comments (if any)

NA

## Which issue(s) does this PR fix?

closes #99

### CODE REVIEW

**Comment by talbright** [comment]
@wadackel / @mavogel can you take a look?

File: chglog.go [hunk 1]
```diff
@@ -324,18 +326,6 @@ func (gen *Generator) render(w io.Writer, unreleased *Unreleased, versions []*Ve
 		"datetime": func(layout string, input time.Time) string {
```
Comment (talbright) [inline chglog.go:326]: We can remove all the funxs here but that would break backwards compatibility. Let me know.

Comment (clok) [reply chglog.go:326]: Right on. I'd like to do some testing to make sure.

**Review by clok — COMMENTED** [review]
Thank you for picking this work up. Left a minor comment. I'd like to test it a bit locally. Hopefully I can get to that tomorrow.

**Review by clok — APPROVED** [review]
Looks good in local testing. I'm excited to get Sprig template function support in the tool!

Thanks again to @delmendo for the original push on this feature and @talbright for the follow up to get it over the line. 🍻

**Comment by clok** [comment]
This has been releases as a part of the [v0.14.0](https://github.com/git-chglog/git-chglog/releases/tag/v0.14.0) release. Thank you for your contribution!

**Pull request merged**